generator.yml

```diff
@@ -900,6 +900,10 @@ builds:
   - {<< : *hexagon_allmod,    << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_latest}
   - {<< : *i386,              << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_latest}
   - {<< : *i386_suse,         << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_latest}
+  - {<< : *loong,             << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_latest}
+  - {<< : *loong_lto_thin,    << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_latest}
+  - {<< : *loong_allyes,      << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_latest}
+  - {<< : *loong_allyes_lto,  << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_latest}
```

Comment:
I know it didn't land in this PR, and I missed this comment on the PR that added it, but we're not testing allyesconfig with LTO for any other arch, right? Why for loongarch?

Reply:
It is only `allyesconfig` because testing LoongArch with `allmodconfig` would not make much sense since we have to disable `CONFIG_MODULES`. Other architectures get tested with `allmodconfig`, which is what LoongArch should switch to once module support has been added to the toolchain.

Reply:
but allmodconfig+lto?

Reply:
Yeah, we test `allmodconfig` with LTO for arm64 and x86_64.